site/src/components/gradients.md

```diff
@@ -17,12 +17,19 @@ There are two main types of gradients.
 
 ## Text color gradients
 
-Currently there is only one available gradient for text. Because gradients transitions take up the entire width of a particular element, it's recommended to highlight inline elements or a portion of a heading, not the entire heading itself.
+Gradients transitions take up the entire width of a particular element, it's recommended to highlight inline elements, icons, or a portion of a heading, not the entire heading itself.
+
+| base class name             | interpolated value     |
+| --------------------------- | ---------------------- |
+| `gradient-text-<colorname>` | `purple-blue`, `vivid` |
 
 ```html
 <h3 class="font-size-h3 font-weight-bold">
-	A gradient from <span class="gradient-text-purple-blue">purple to blue</span>
-</h2>
+	A dark gradient from <span class="gradient-text-purple-blue">purple to blue</span>
```

Comment:
This gradient is theme-aware, and I'm not sure "dark" quite applies in all themes. Maybe "muted"?

```suggestion
	A muted gradient from <span class="gradient-text-purple-blue">purple to blue</span>
```

Reply:
Makes sense, updated.